vaadin-archetype-application/src/main/resources/archetype-resources/LICENSE.md

```diff
@@ -0,0 +1,24 @@
+This is free and unencumbered software released into the public domain.
```

Comment:
I guess this was also copy pasted from viritin flow archetypes. I don't know what licenses archetypes usually put for the generated code if any. At least this should be fine as users can do what they want with this, even change the license.

Reply:
Unlicensed should be used for any project templates like starters and archetypes.